src/cmd/run.go

```diff
@@ -63,13 +75,69 @@ var runCmd = &cobra.Command{
 			message.Fatalf(err, "Cannot unmarshal %s", config.TaskFileLocation)
 		}
 
-		if config.ListTasks {
+		ListTasks = cmd.Flag("list").Value.String() == "true"
+		ListAllTasks = cmd.Flag("list-all").Value.String() == "true"
```

Comment:
```suggestion
```
Can kill these 2 lines because the flag already populated the vars

Reply:
nice!